sdk/attestation/azure-security-attestation/samples/attestation/attest_openenclave_with_draft_policy.cpp

```diff
@@ -10,14 +10,6 @@
  * - ATTESTATION_ISOLATED_URL:  Points to an Attestation Service Instance in Isolated mode.
  * - LOCATION_SHORT_NAME:  Specifies the short name of an Azure region to use for shared mode
  * operations.
- * - AZURE_TENANT_ID:     Tenant ID for the Azure account. Used for authenticated calls to the
- * attestation service.
- * - AZURE_CLIENT_ID:     The Client ID to authenticate the request. Used for authenticated calls to
```

Comment:
It might be useful to keep these, since the samples won't work without them. Not 100% sure though.

Reply:
Sample will work, if user has Azure CLI installed, for example.
I will create a separate PR with the same brief paragraph about using DefaultAzureCredential, and will apply it in all places.

Reply:
#5205